### PR TITLE
Update tf_sig_build_dockerfile README and rename ROCM script

### PR DESCRIPTION
**Pull request by jayfurmanek**
Updates the README to reference published build containers at `rocm/tensorflow-build`
This also updates the rename script to be able to handle tf_nightly_rocm => tf-nightly-rocm

### CODE REVIEW

File: tensorflow/tools/tf_sig_build_dockerfiles/README.md [hunk 1]
```diff
@@ -49,7 +49,8 @@ DOCKER_BUILDKIT=1 docker build \
 ```
 For ROCM
 ```
-docker build -f Dockerfile.rocm --build-arg ROCM_VERSION=5.3.0 --build-arg PYTHON_VERSION=3.10 -t my-tf-devel . 
+DOCKER_BUILDKIT=1docker build -f Dockerfile.rocm \
```
Comment (micmelesse) [inline tensorflow/tools/tf_sig_build_dockerfiles/README.md]: there should be a space between the 1 and docker

**Review by micmelesse — DISMISSED** [review]

File: tensorflow/tools/tf_sig_build_dockerfiles/README.md (hunk 1 above)
Comment (jayfurmanek) [reply tensorflow/tools/tf_sig_build_dockerfiles/README.md]: oops, thx. good catch

File: tensorflow/tools/tf_sig_build_dockerfiles/README.md [hunk 2]
```diff
@@ -102,19 +103,36 @@ Now let's build `tf-nightly`.
...
     - `tensorflow/build:latest-python3.9`
     - `tensorflow/build:latest-python3.8`
     - `tensorflow/build:latest-python3.7`
 
+For ROCM
+
+    - `rocm/tensorflow-build:latest-python3.10`
+    - `rocm/tensorflow-build:latest-python3.9`
+    - `rocm/tensorflow-build:latest-python3.8`
+    - `rocm/tensorflow-build:latest-python3.7`
```
Comment (micmelesse) [inline tensorflow/tools/tf_sig_build_dockerfiles/README.md:118]: Are these Centos based images?

**Review by micmelesse — COMMENTED** [review]
If you have tested these changes and everything is good then LGTM.

File: tensorflow/tools/tf_sig_build_dockerfiles/README.md (hunk 2 above)
Comment (jayfurmanek) [reply tensorflow/tools/tf_sig_build_dockerfiles/README.md:118]: yes, these are built with the new manylinux dockerfile and pushed to dockerhub

Comment (micmelesse) [reply tensorflow/tools/tf_sig_build_dockerfiles/README.md:118]: Great, Was not sure.

**Comment by jayfurmanek** [comment]
yes, I have tested. Can you approve? Thx!

**Review by micmelesse — APPROVED** [review]

**Pull request merged**